source/config/profileUpgradeSteps.py

```diff
@@ -99,3 +99,16 @@ def upgradeConfigFrom_5_to_6(profile: dict):
 	if useInMSWord:
 		from . import AllowUiaInMSWord
 		profile['UIA']['allowInMSWord'] = AllowUiaInMSWord.ALWAYS.value
+
+
+def upgradeConfigFrom_6_to_7(profile: dict):

```

Comment:
While inconsistent with other function definitions, I believe this typing is more accurate

```suggestion
def upgradeConfigFrom_6_to_7(profile: Dict[str, str]) -> None:
```